lms/djangoapps/ccx/views.py

```diff
@@ -57,6 +58,8 @@
 
 log = logging.getLogger(__name__)
 TODAY = datetime.datetime.today  # for patching in tests
+# this is an arbitrary hard limit
+CCX_MAX_STUDENT_ALLOWED = 200
```

Comment:
Maybe this should be a configurable setting pulled in from the json using `aws.py`, with the default of 200 set in `common.py` (which is included by `aws.py`).